biomage/utils/data.py

```diff
@@ -18,13 +77,57 @@ def modified_records(item, target_table, config):
             "meta": {
```

Comment:
I think you are overusing this ** expansion thing. This whole thing for staging experiments is as simple as:
```
cfg["projectId"]["S"] = f"{extra['sandbox_id']}-{item['projectId']['S']}"
cfg["M"]["gems"]["M"]["paramsHash"]["S"] = ""
cfg = add_env_user_to_experiment(cfg)
return cfg

```